OpenRA.Mods.Common/Widgets/WidgetUtils.cs

```diff
@@ -232,7 +232,7 @@ public static string WrapText(string text, int width, SpriteFont font)
 			var textSize = font.Measure(text);
 			if (textSize.X > width)
 			{
-				var lines = text.Split('\n').ToList();
+				var lines = text.Replace("\r", "").Split('\n').ToList();
```

Comment:
```suggestion
				var lines = text.Replace("\n\r", "\n").Replace("\r", "\n").Split('\n').ToList();
```

Also it should be like this

Reply:
Why is that necessary? A `\r` on its own doesn't make sense to me.

Reply:
windows doesn't make much sense yes

Reply:
? The new lines are `\r\n`. After splitting on `\n` the `\r`s remain alone, hence we remove them here...

Reply:
new lines can also be just \r on windows